### DISCUSSION TITLE
Explain how tokens are more secure than username/password.

### DISCUSSION THREAD

**Discussion by mevenson** [community/community#16036] [Other Feature Feedback, Questions, & Ideas]
I still cannot fathom how using a token is more secure that username/password authentication. I can remember my user name and my password – Try remembering one of those tokens. The only way to be able to use it is to save it a file somewhere or type it in every time. How in the world is that more secure? With username and password authentication, you have to know two things. With a token – well you get the picture. And with the token, I have to have the file on EVERY computer that I use to access github and I use a lot of them. This whole business of not allowing username/password authentication from applications is simply ludicrous. It’s like you got a new hammer and now you want to throw out all of the other tools in their tool box and use the hammer for everything. I say fine – allow token authentication for those who think they can benefit from it, but don’t get rid of username/password. You guys have just made using github more difficult and frustrating.

**Comment by Secret-chest** [comment]
Please.
